scripts/scil_apply_transform_to_image.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+    Transform *.nii or *.nii.gz using an affine/rigid transformation.
+
+    For more informations on how to use the various registration scripts
```

Comment:
Please remove this sentence.

Reply:
I wanted to ask about this, do we transfert documention/example ?

Reply:
I think we should provide as much information as possible especially because everything is public. @mdesco ?